tests/conftest.py

```diff
@@ -29,7 +29,7 @@
 from sagemaker.sklearn.defaults import SKLEARN_VERSION
 from sagemaker.tensorflow.estimator import TensorFlow
 
-DEFAULT_REGION = "us-west-2"
+DEFAULT_REGION = "us-east-1" if os.getenv("DLC_GAMMA_TESTING") else "us-west-2"
```

Comment:
this feels like a strange thing to hardcode here. can the DLC gamma tests not pass the region in with the test invocation?